CONTRIBUTING.md

```diff
@@ -79,6 +86,15 @@ TODO
 
 The test suite must pass before merging the PR to our main branch. Any new feature, change or fix must be covered by corresponding tests.
 
+#### Tests with dependencies
+
+Some tests, e.g. the [PAM CLI tests](https://github.com/ubuntu/authd/blob/5ba54c0a573f34e99782fe624b090ab229798fc3/pam/integration-tests/integration_test.go#L21), use external tools such as [vhs](https://github.com/charmbracelet/vhs)
+to record and run the tape files needed for the tests. Those tools are not included in the project dependencies and must be installed manually.
+
+For more information about their usage, refer to the following documentations:
+
+- [vhs](https://github.com/charmbracelet/vhs?tab=readme-ov-file#tutorial)
```

Comment:
This is fine for defining the .tape file, but is there anything else that should be documented to re-generate the reference (golden) files on changes?

Reply:
This is done by running `go test` with the `--update` flag, but since it's a general concept that's used in the entire project, we should probably write about it when we decide to write the final documentation.